spec/brcobranca/core_ext_spec.rb

```diff
@@ -174,6 +174,20 @@ def modulo_11_de_2_ate_9(numero)
     it { expect { modulo_11_de_2_ate_9 '2582fd81' }.to raise_error(ArgumentError) }
   end
 
+  # Ex: Sicoob
+  describe 'Módulo 11 com multiplicadores 3, 7, 9, 1, trocando 11 e 10 por 0' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.